### ISSUE TITLE
Flipper Android 0.71

### ISSUE THREAD

**Issue by billnbell** [facebook/react-native#35834]
### Description

FLIPPER_VERSION=0.176.1

I am getting null in Debug mode and addPlugin fails. Why?

Also we should add a check for null.

```
 public static void initializeFlipper(Context context, ReactInstanceManager reactInstanceManager) {
    if (FlipperUtils.shouldEnableFlipper(context)) {
      final FlipperClient client = AndroidFlipperClient.getInstance(context);
      if (client == null) {
        return;
      }
```

But I have flipper running in desktop and I keep getting null. How do I figure out the issue?

### Version

0.71

### Output of `npx react-native info`

None

### Steps to reproduce

None

### Snack, code example, screenshot, or link to a repository

None

**Comment by togenius1** [comment]
same error

**Comment by billnbell** [comment]
??? This seems like a big issue.

**Comment by billnbell** [comment]
Do we need to install `react-native-flipper` ? I don't see anything about that in the docs?

**Comment by cortinico** [comment]
> I am getting null in Debug mode and addPlugin fails. Why?

Can you clarify what's your issue more exactly?
Where are you getting `null`?
Is Flipper workign fine on a empty project, without bumping to `0.176.1`?

> Do we need to install react-native-flipper ? I don't see anything about that in the docs?

Nope you don't need to install `react-native-flipper`

**Comment by billnbell** [comment]
I put the fix in above. The client returns null for me.

```
final FlipperClient client = AndroidFlipperClient.getInstance(context);
```

This is null!

**Comment by cortinico** [comment]
Is Flipper workign fine on a empty project, without bumping to 0.176.1?

**Comment by billnbell** [comment]
I cannot get it to work at all. 0.125 or 0.176 both do not work for me. Yea I did clean and run rm Podfile.lock && pod deintegrate && pod install between runs... any ideas?

**Comment by cortinico** [comment]
> Yea I did clean and run rm Podfile.lock && pod deintegrate && pod install between runs... any ideas?

Initially you mentioned this is an Android issue. Here you mention `pod install` so it's unclear to me what's the issue you're having. Is it Android or iOS?

**Comment by billnbell** [comment]
you are right. I am losing my mind. It is 100% Android.

I am trying 0.176 - how do I make sure that is set for Android

**Comment by cortinico** [comment]
> I am trying 0.176 - how do I make sure that is set for Android

My question was if the issue is happening on version 0.125.0 (the version shipped with RN 0.71).
